config/environments/development.rb

```diff
@@ -1,54 +1,23 @@
 Rails.application.configure do
-  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
Making changes like this just add too much clutter in the diff of the PR and makes it hard to review actual changes that are relevant to the PR itself. I get that you may have deleted these comments in response to a previous comment about having too many changes in the PR, but you have to understand that these comments were not counted as part of the changes of this PR but now they are because you removed them. 

Will let this slide for now but in future only modify files that directly have an impact on your feature

Reply:
Roger that :)